app/src/main/java/de/westnordost/streetcomplete/quests/sport/Sport.kt

```diff
@@ -0,0 +1,47 @@
+package de.westnordost.streetcomplete.quests.sport
+
+enum class Sport(val osmValue: String) {
+    // sorted by ~worldwide usages, minus country specific ones
+    MULTI("multi"),
+
+    // 250k - 10k
```

Comment:
the comments about the usage numbers should actually go to the quest form, because this is where the order matters. The order does not matter in an enum (read: should not)

Reply:
But I can just do that after the merge

Reply:
In the quest form, the list is sorted by country usage anyway, so order there shouldn't matter either.

Reply:
only the top few